src/formatter.rs

```diff
@@ -13,6 +14,34 @@ use zksync_types::{vm_trace::Call, StorageLogQuery, StorageLogQueryType, VmEvent
 
 use lazy_static::lazy_static;
 
+#[derive(Debug, clap::Parser, Clone, clap::ValueEnum, PartialEq, Eq)]
+pub enum ShowCalls {
+    None,
+    User,
+    System,
+    All,
+}
+
+impl FromStr for ShowCalls {
+    type Err = ();
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s.to_lowercase().as_ref() {
+            "none" => Ok(ShowCalls::None),
+            "user" => Ok(ShowCalls::User),
+            "system" => Ok(ShowCalls::System),
+            "all" => Ok(ShowCalls::All),
+            _ => Err(()),
```

Comment:
A little of a vague error, would be better to add some description

Reply:
good point. Done